### PR TITLE
Documentation on: ASSET_EVENTS_ACTIONS_EXTENSION

### PR DESCRIPTION
**Pull request by ajerard93**
This is adding documentation about the new iris -app sdk extension called ASSET_EVENTS_ACTIONS_EXTENSION (link to the PR [here](https://github.com/Trackunit/manager/pull/9388))

### CODE REVIEW

File: sdk-docs/getting-started/extension-points.md [hunk 1]
```diff
@@ -5,44 +5,45 @@ parentDocSlug: getting-started
...
+{
+"image": [
+"https://cdn.statically.io/gh/trackunit/developer-hub/master/sdk-docs/extension_to_asset_home.png",
+null,
+"Asset home extension"
+],
+"align": "center",
+"sizing": "50% "
+}
+]
```
Comment (kla-trackunit) [inline sdk-docs/getting-started/extension-points.md:33]: Any reason to de indent this?

File: sdk-docs/getting-started/extension-points.md [hunk 2]
```diff
@@ -52,49 +53,46 @@ conditions: {
...
+{
+"image": [
+"https://cdn.statically.io/gh/Trackunit/developer-hub/master/sdk-docs/extension_to_site.png",
+null,
+"Site home extension"
+],
+"align": "center",
+"sizing": "50% "
+}
+]
```
Comment (kla-trackunit) [inline sdk-docs/getting-started/extension-points.md:72]: Any reason to de indent this?

File: sdk-docs/getting-started/extension-points.md [hunk 3]
```diff
@@ -52,49 +53,46 @@ conditions: {
...
+{
+"image": [
+"https://cdn.statically.io/gh/Trackunit/developer-hub/master/sdk-docs/extension_to_navbar.png",
+null,
+"Fleet wide extension"
+],
+"align": "center",
+"sizing": "50% "
+}
+]
```
Comment (kla-trackunit) [inline sdk-docs/getting-started/extension-points.md:92]: Any reason to de indent this?

File: sdk-docs/getting-started/extension-points.md [hunk 4]
```diff
@@ -112,59 +110,57 @@ The Report Extension Point allows you to add a new report to the Reports screen
...
+{
+"image": [
+"https://cdn.statically.io/gh/Trackunit/developer-hub/master/sdk-docs/extension_reports.png",
+null,
+"Report extension"
+],
+"align": "center",
+"sizing": "50% "
+}
+]
```
Comment (kla-trackunit) [inline sdk-docs/getting-started/extension-points.md:123]: Any reason to de indent this?

File: sdk-docs/getting-started/extension-points.md [hunk 5]
```diff
@@ -196,3 +192,24 @@ gridOptions: {
 }
 ```
 
+# Asset Events Actions extension point
+
+This extension point allows you to add a user interface in the Events within Asset Home in Trackunit Manager. The extension is controlled in the extension-manifest.ts file.
+
```
Comment (kla-trackunit) [inline sdk-docs/getting-started/extension-points.md:198]: An image for this would be great! 
But it can also wait :)

**Review by kla-trackunit — APPROVED** [review]

File: sdk-docs/getting-started/extension-points.md (hunk 1 above)
Comment (ajerard93) [reply sdk-docs/getting-started/extension-points.md]: no reason, looks like its a sneaky work of my formatter (i think? ). I have added back all the indentation now :)

File: sdk-docs/getting-started/extension-points.md (hunk 5 above)
Comment (ajerard93) [reply sdk-docs/getting-started/extension-points.md:198]: I agree, Thank you. My plan to add an image once my extension PR gets approved :)

**Review by kla-trackunit — APPROVED** [review]

**Pull request merged**